tests/Cms/App/AppTest.php

```diff
@@ -709,4 +709,45 @@ public function testVersionHash()
     {
         $this->assertEquals(md5(App::version()), App::versionHash());
     }
+
+    public function testSlugsOption()
+    {
+        // string option
+        new App([
+            'roots' => [
+                'index' => '/dev/null'
+            ],
+            'options' => [
+                'slugs' => 'de'
+            ]
+        ]);
+
+        $this->assertSame('AE', Str::$language['Ä']);
+
+        // string option with dot notation
+        new App([
+            'roots' => [
+                'index' => '/dev/null'
+            ],
+            'options' => [
+                'slugs.language' => 'de'
```

Comment:
This second test should use a different language code to make sure that the first test isn't the reason why this passes.